data/human/deep missions.txt

```diff
@@ -2328,15 +2328,21 @@ mission "Deep: Scientist Rescue 2: Pirate Bactrian"
 	to offer
 		has "Deep: Scientist Rescue 1: done"
 	to fail
-		has "Deep: Scientist Rescue 2: done"
-	npc kill
+		or
+			has "Deep: Scientist Rescue 2: done"
+			has "deep: cosmic devil destroyed early"
+	to complete
+		never
+	npc
 		government "Pirate"
 		personality staying nemesis
 		ship "Bactrian (Cosmic Devil)" "Cosmic Devil"
-		dialog
-			`After the Bactrian is eliminated, you receive a transmission from an unknown source. It suddenly opens on its own and an electronically distorted voice begins speaking to you.`
-			`	"Well done, Captain. Well done at being a thorn in my side. You will pay for this. One day."`
-			`The transmission cuts out as abruptly as it had started.`
+		on kill
```

Comment:
I've not tested this before, but would leaving the Bactrian disabled in the system result in a duplicate Bactrian being spawned in the subsequent mission? Maybe you should have an `on disable` block that still sets the flag but prompts the player to go finish the job?

Reply:
I don't think this is an issue, as the mission containing the disabled variant would fail. If that caused a duplicate bactrian, ignoring it entirely would also spawn a duplicate.